test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
@okomarov The problem with `isHG2()` is that it might not be present in older versions of MATLAB and Octave. So to be able to skip the test, one needs to check whether that function exists (otherwise an error is thrown), i.e. a catch-22.

An alternative approach, could be to have a `try-catch` inside the test and explicitly catch the error code for "function not found" (or we could even introduce that into the test handler).